dockers/cemba/vcftoallc/convert-vcf-to-allc.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+import argparse
+import gzip
+import re
+
+COMMENT = re.compile(r"^#")
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+      "--input_vcf", "-i", dest="input_vcf", default=None, help="input VCF"
+    )
+    parser.add_argument(
+      "--output_allc", "-o", dest="output_allc", default=None,
+      help="output ALLC"
+    )
+    args = parser.parse_args()
+
+    writeBinary = False
+    if args.input_vcf:
+        with gzip.open(args.input_vcf, "rt") if args.input_vcf.endswith(
+            ".gz"
+        ) else open(args.input_vcf, "r") as vcf_file:
+            with gzip.open(args.output_allc, "wb") if args.output_allc.endswith(
+                ".gz"
+            ) else open(args.output_allc, "w") as allc_file:
+
+                writeBinary = args.output_allc.endswith(".gz")
+
+                for line in vcf_file:
+                    if COMMENT.search(line):
```

Comment:
rather than defining a regular expression, you could just use the python functions `startswith()`
```suggestion
                    if line.startswith("#"):
```

Reply:
Changed!